bootstrap.sh

```diff
@@ -20,24 +20,29 @@ fi
 if [[ $(id -g) = 0 ]]; then
   echo "Cannot run as root, defaulting to GID 1000"
   export GROUP_ID=1000
-elif [[ $(uname) == "Darwin" ]]; then
+elif [[ $(uname) == "Darwin" ]] ; then
```

Comment:
Can we change this to not normalise for Darwin and instead spit an error suggesting that the Development workflow does not work for macOS?